.devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:16",
+  "hostRequirements": {
+    "cpus": 4,
+    "memory": "8gb",
+    "storage": "32gb"
```

Comment:
What will happen if I can not provide such requirements?

Reply:
That's for Codespaces, not for your computer. 😅

Reply:
https://code.visualstudio.com/docs/remote/devcontainerjson-reference#_minimum-host-requirements

Reply:
I authorize you to develop on an Intel 80386.

Reply:
I can run it still locally right?

Reply:
Ok so it just displays a warning...

> I authorize you to develop on an Intel 80386.

If you need such an amount of resources to dev CML...

Reply:
> _it just displays a warning..._

Yes, nothing else

Reply:
> _If you need such an amount of resources to dev CML..._

Codespaces machines with less than that are too slow. 🐢

Reply:
I do not pretend to use Codespaces my connection sometimes is not great

Reply:
But I do 🙃